### PR TITLE
use fake shoeboxClient in test. remove unnecessary codes.

### PR DESCRIPTION
**Pull request by yingjieMiao**

### CODE REVIEW

File: shoebox/test/com/keepit/search/graph/URIGraphTest.scala [hunk 1]
```diff
@@ -366,12 +336,7 @@ class URIGraphTest extends Specification with DbRepos {
     }
 
     "search personal bookmark domains" in {
-       running(new DevApplication().withFakeHttpClient.withShoeboxServiceModule
-          .overrideWith(new FortyTwoModule {
-            override def configure() {
-              bind[HttpClient].toInstance(new FakeHttpClient(Some({case s => Json.stringify(httpClientGetChangedUsers(s.split("=")(1).toLong))})))
-            }
-          })) {
+       running(new DevApplication().withShoeboxServiceModule) {
```
Comment (andrewconner) [inline shoebox/test/com/keepit/search/graph/URIGraphTest.scala:339]: Do all of these need to be `DevApplication`s? You should be able to provide the modules it needs on top of an EmptyApplication.

File: shoebox/test/com/keepit/shoebox/FakeShoeboxServiceClientImpl.scala [hunk 2]
```diff
@@ -0,0 +1,96 @@
+package com.keepit.shoebox
```
Comment (andrewconner) [inline shoebox/test/com/keepit/shoebox/FakeShoeboxServiceClientImpl.scala:1]: Great, much better location for this :)

File: shoebox/test/com/keepit/search/graph/URIGraphTest.scala (hunk 1 above)
Comment (yingjieMiao) [inline shoebox/test/com/keepit/search/graph/URIGraphTest.scala:339]: I tried to switch to emptyApp, failed test

Comment (andrewconner) [inline shoebox/test/com/keepit/search/graph/URIGraphTest.scala:339]: It may need other modules added on (like withFakePersistEvent() and such).

Comment (yingjieMiao) [inline shoebox/test/com/keepit/search/graph/URIGraphTest.scala:339]: OK. I can try it later.

**Pull request merged**